src/components/Icons/CongestionChargeAppliesIcon.tsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import Icon from "@/components/Icons/Icon";
+import { faCopyright } from "@fortawesome/free-solid-svg-icons";
+
+const CongestionChargeAppliedIcon: React.FC = () => {
```

Comment:
Can a propsless Functional Component have the type parameter `React.FC<{}>` to make it explicitly clear? 🙂